fe/fe-core/src/test/java/org/apache/doris/nereids/trees/expressions/ExpressionParserTest.java

```diff
@@ -25,18 +25,18 @@
 public class ExpressionParserTest {
     private static final SqlParser PARSER = new SqlParser();
 
-    private void assertSql(String sql) throws Exception {
+    private void assertSql(String sql) {
         TreeNode treeNode = PARSER.parse(sql);
-        System.out.println(treeNode.toString());
+        assert treeNode != null;
```

Comment:
if you want to assert, it is better to use junit functions

Reply:
done